docs/handbook/image-file-formats.rst

```diff
@@ -88,8 +88,14 @@ in ``L``, ``RGB`` and ``CMYK`` modes.
 Loading
 ~~~~~~~
 
+To use Ghostscript, Pillow searches for the "gs" executable. On Windows, it
+also searches for "gswin32c" and "gswin64c". If you would like to customise
```

Comment:
Bit more concise:

```suggestion
also searches for "gswin32c" and "gswin64c". To customise
```

Reply:
Sure. I've pushed a commit so that I could rearrange the lines ending as well, and shortened another bit of text later on.